src/test/java/hw2/Exercise2.java

```diff
@@ -0,0 +1,88 @@
+package hw2;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.testng.Assert.*;
+
+public class Exercise2 extends BaseTest {
+
+    @Test
+    public void testWebDriverHardAsserts() {
+
+        //Open test site by URL
+        String openedURL = openSite();
+        assertEquals(openedURL, URL);
+
+        //Assert Browser title
+        assertTrue(homePageIsOpened());
+
+        //Perform login
+        assertTrue(login());
+
+        //Assert User name in the left-top side of screen that user is loggined
+        assertTrue(checkUsername());
+
+        //Open through the header menu Service -> Different Elements Page
+        driver.findElement(By.linkText("Service")).click();
+        driver.findElement(By.linkText("Different elements")).click();
+        assertEquals(driver.getCurrentUrl(), "https://jdi-testing.github.io/jdi-light/different-elements.html");
+
+        //Select checkboxes
+        List<WebElement> checkboxes = driver.findElements(By.cssSelector("label.label-checkbox"));
+        for (WebElement element : checkboxes) {
+            if (element.getText().equals("Wind") || element.getText().equals("Water")) {
+                element.click();
+            }
+        }
```

Comment:
1. Why do you decide do not send as parameter name of the checkbox?
2. It is better to swap places String and element to avoid `NPE`